k8s-opensearch/gke_sysctl_values.yaml

```diff
@@ -0,0 +1,24 @@
+ # workaround from https://github.com/opensearch-project/helm-charts/issues/87
+ extraInitContainers: 
+   - name: sysctl
+     image: docker.io/bitnami/bitnami-shell:10-debian-10-r199
+     imagePullPolicy: "IfNotPresent"
+     command:
+       - /bin/bash
+       - -ec
+       - |
+         CURRENT=`sysctl -n vm.max_map_count`;
+         DESIRED="262144";
+         if [ "$DESIRED" -gt "$CURRENT" ]; then
+             sysctl -w vm.max_map_count=262144;
+         fi;
+         CURRENT=`sysctl -n fs.file-max`;
+         DESIRED="65536";
+         if [ "$DESIRED" -gt "$CURRENT" ]; then
+             sysctl -w fs.file-max=65536;
+         fi;
+     securityContext:
+       runAsUser: 0
+       # this is no-bueno, but seems the only way to get this to work in GKE...
+       # https://github.com/opensearch-project/helm-charts/issues/87#issuecomment-982185661
+       privileged: true
```

Comment:
Yeah, lets move this into the locals and do a:
```
extraInitContainers = [
  securityContext = {
    privileged = var.kubernetes_cluster.specs.kubernetes.distribution == "gke" ? true : false
  }
]
```
If that works...not sure how the yaml merging will work with an array of objects.

EDIT: Ohhh...I see what you're doing here... Did you set this up as a separate file cuz the local merging of yaml arrays of objects wasn't working?

Reply:
No strong feelings. I think yaml is definitely easier to read than a set of HCL locals, this is just a new pattern we haven't needed before.